lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -709,6 +709,9 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, layerID string, imageStore string) *containers.CreateParams {
 	config := &models.ContainerCreateConfig{}
 
+	config.NumCpus = &cc.HostConfig.CPUCount
+	config.Memory = &cc.HostConfig.Memory
```

Comment:
Update: now that I've changed it, I've noticed that VC doesn't really recognize things in MB/GB etc. If I create a VM with 1000MB, VC shows it as 1000MB of memory ... however, if I create one with 1024MB, VC shows it as 1GB of memory. Additionally, if I create a VM with `1GB` of memory, the VMX shows `memSize` to be `1024`.